site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -19,6 +19,11 @@ True if CleanSpeak username moderation is enabled.
 [field]#application.data# [type]#[Object]# [optional]#Optional#::
 An object that can hold any information about the Application that should be persisted.
 
+[field]#application.formConfiguration.adminRegistrationFormId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.20.0#::
+The unique Id of the form to use for the Add and Edit User Registration form when used in the FusionAuth admin UI. 
++
+A paid edition of FusionAuth is required to utilize custom forms. When this parameter is not provided it will default to a form provided by FusionAuth that may be used without a paid edition.
```

Comment:
Review the code path that sets this default, it will be defaulted to a specific form that is in use by the FusionAuth application. 

See other examples in the docs of where we describe the default value in the description to keep the correct formatting.